examples/basic-video.html

```diff
@@ -0,0 +1,101 @@
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Dev Example — Networked-Aframe</title>
+    <meta name="description" content="Dev Example — Networked-Aframe">
+
+    <script src="https://aframe.io/releases/1.2.0/aframe.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/socket.io/2.3.0/socket.io.slim.js"></script>
+    <script src="/easyrtc/easyrtc.js"></script>
+    <script src="/dist/networked-aframe.js"></script>
+
+    <script src="https://unpkg.com/aframe-randomizer-components@^3.0.1/dist/aframe-randomizer-components.min.js"></script>
+    <script src="https://unpkg.com/aframe-particle-system-component@1.0.5/dist/aframe-particle-system-component.min.js"></script>
+    <script src="/js/spawn-in-circle.component.js"></script>
+  </head>
+  <body>
+    <a-scene networked-scene="
+      room: basic-video;
+      debug: true;
+      adapter: easyrtc;
+      audio: false;
+      video: true;
```

Comment:
Actually I was more leaning to remove the audio and video options from networked-scene, see discussion in https://github.com/networked-aframe/networked-aframe/issues/80#issuecomment-360659707

Reply:
Well, i think that important to be able to do audio or/and video only.
Can we do that in a separate PR?

Reply:
That's fine with me.